config/webpack.js

```diff
@@ -0,0 +1,69 @@
+// This file is based on Gitlab's config/webpack.config.js file.
+
+const path = require('path');
+const webpack = require('webpack');
+const CompressionPlugin = require('compression-webpack-plugin');
+
+const ROOT_PATH = path.resolve(__dirname, '..');
+const IS_PRODUCTION = process.env.RAILS_ENV === 'production';
+
+var config = {
+  context: path.join(ROOT_PATH, 'app/assets/javascripts'),
+
+  entry: {
+    application: './application.js',
+  },
+
+  output: {
+    path: path.join(ROOT_PATH, 'public/assets/webpack'),
+    publicPath: '/assets/webpack/',
+    filename: IS_PRODUCTION ? '[name]-[chunkhash].js' : '[name].js',
+  },
+
+  plugins: [],
```

Comment:
Missing [StatsPlugin](https://github.com/mipearson/webpack-rails/blob/master/example/webpack.config.js#L35) to generate manifest.json that will be necessary in order to `webpack_assets_path` helper work. At least on development, I didn't test for production.